charts/argo-workflows/values-ha.yaml

```diff
@@ -0,0 +1,23 @@
+# Sample values for High Availability configuration, following https://argoproj.github.io/argo-workflows/high-availability/
```

Comment:
I don't like this approach to have multiple values.yaml inside the chart root. But we can follow the approach from Argo CD for sure:
- Place a section into the Charts README
- move this file into `ci/`. This ensures, that our CI workflow tests with these values

Reply:
FYI, I believe it has to end in -values.yaml for the ci to pick it up (eg ha-values.yaml).

Reply:
Ah did not realize Argo CD had a section on this. I can make those changes.
Would probably prefer to link from the a new README section to the `ci/` files instead of having a [duplicate code block](https://github.com/argoproj/argo-helm/blob/962342fe2acef6022ac5c1a3eb352b336308b3eb/charts/argo-cd/README.md?plain=1#L27) in the docs though.

> FYI, I believe it has to end in -values.yaml for the ci to pick it up (eg ha-values.yaml).

I was wondering how these got picked up, so great note!

Reply:
Updated as per above